heartbeat/monitors/active/http/check.go

```diff
@@ -132,14 +134,100 @@ func checkHeaders(headers map[string]string) respValidator {
 	}
 }
 
-func checkBody(matcher []match.Matcher) bodyValidator {
-	return func(r *http.Response, body string) error {
-		for _, m := range matcher {
-			if m.MatchString(body) {
+func parseBody(b interface{}) ([]match.Matcher, []match.Matcher) {
+	var (
+		pm, nm []match.Matcher
+		positive = "positive"
```

Comment:
Why are `positive` and `negative` declared as vars here? It's a bit odd since they are only used once. Can we switch to just using string literals later.

Reply:
Sure, will adjust them.